sw/device/lib/testing/spi_device_testutils.h

```diff
@@ -35,6 +35,7 @@ typedef enum spi_device_flash_opcode {
   kSpiDeviceFlashOpBlockErase32k = 0x52,
   kSpiDeviceFlashOpBlockErase64k = 0xd8,
   kSpiDeviceFlashOpPageProgram = 0x02,
+  kSpiDeviceFlashOpPageQuadProgram = 0x32,
```

Comment:
I feel more iffy about putting this in the list here. This opcode varies considerably across parts (for example, a particular Winbond part had it as 0x32, and a Macronix part had it as 0x38). Perhaps consider adding an opcode argument to `spi_flash_testutils_program_page_quad`?

Reply:
Thanks for raising this. I removed it from this file and made the function `spi_flash_testutils_program_page_quad` flexible by receiving the opcode.

Reply:
Thanks!